packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carbon/ibmdotcom-web-components",
-  "version": "0.8.0-rc.0",
+  "version": "1.15.0-rc.0",
```

Comment:
Thank you for updating @asudoh!

We should actually bump to `1.0.0-rc.0`, right?

Reply:
Yeah wondering which was better, will update. Thanks @jeffchew for your feedback!